README.md

```diff
@@ -338,6 +339,13 @@ const onBlur = ({width, height, x, y, top, left, node}, {prop1, prop2}, {event,
 ...
 ```
 
+
+##### `autoRestoreFocus`: boolean
```

Comment:
Please move this info ####Config section instead (since it is also possible to put this as config variable passed to withFocusable({...})).
Then in this section you can put the same as we do with other props like:
```
### `autoRestoreFocus `: boolean
Same as in [config](#config).
```